docs[Op#50983] Add system admin and user guides for OneDrive/Sharepoint

akabiru opened the pull request:
https://community.openproject.org/work_packages/50983

Define a **_placeholder_** for the new OneDrive/Sharepoint integration documentation- currently assigned to Op Product

akabiru committed:
docs[Op#50983] Add system admin and user guides for OneDrive/Sharepoint [Draft]

https://community.openproject.org/work_packages/50983

Define a blueprint for new OneDrive/Sharepoint integration

MayaBerd committed:
adding page structure to OneDrive integration documentation

MayaBerd committed:
onedrive documentation update

MayaBerd committed:
adding more file storages documentation structure

akabiru committed:
Merge branch 'dev' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint

akabiru committed:
chore[Op#50983]: update one drive docs link

MayaBerd committed:
Merge branch 'dev' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint

MayaBerd committed:
changing headings structure

MayaBerd committed:
Merge branch 'dev' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint

MayaBerd committed:
Adding DriveID Guide file structure to sharepoint documentation

MayaBerd committed:
Adding Enterprise add-on notes to OneDrive/SharePoint

MayaBerd committed:
Sharepoint docs update

birthe commented on docs/user-guide/file-management/README.md:
```diff
@@ -0,0 +1,42 @@
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Manual upload](#manual-upload)                              | How to manually upload files to work packages in OpenProject. |
+| [Nextcloud integration](#nextcloud-integration)              | How to manage files using Nextcloud integration in OpenProject. |
+| [OneDrive/SharePoint integration](#onedrive/sharepoint-integration-enterprise-add-on) | How to manage files using OneDrive/Sharepoint integration in OpenProject. |
```
Does the "/" work in the title? @as-op ?

as-op replied:
"/" does work in the title, but not in the #anchor link. It is just removed so please link to `#onedrivesharepoint-integration-enterprise-add-on`

MayaBerd committed:
Adding Drive ID guid to Sharepoint documentation

Adding a guide on how to obtain a drive ID guide

MayaBerd committed:
Adding more content to Sharepoint user guide

MayaBerd committed:
Editing sharepoint documentation

Kharonus committed:
[#50983] technical review and small changes

- using one-drive in web links instead of onedrive
- corrected some technical wrongs

Kharonus commented:
Ahoi @as-op ,

I just updated this branch with my review from technical perspective. I had to alter some of the links and fix some previously moved ones. Can you check, if there are still some dead ones? Especially some of the TOC links in some readmes are displayed as not reachable in my IDE.

as-op committed:
Merge branch 'release/13.1' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint

as-op committed:
docs(links): adjust anchors and links to moved pages

as-op commented:
Hi @Kharonus The linkchecker catched them all. I fixed them.

MayaBerd committed:
Merge branch 'release/13.1' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint

MayaBerd committed:
Final changes to OneDrive/SharePoint documentation

removed placeholder text

MayaBerd committed:
Merge branch 'release/13.1' into documentation/50983-add-system-admin-and-user-guides-for-onedrivesharepoint
